src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -37,8 +47,7 @@ class DialogFullScreen extends Modal {
    */
   get dialogClasses() {
     return classNames(
-      'carbon-dialog-full-screen__dialog',
-      { 'carbon-dialog-full-screen__dialog--open': this.props.open }
```

Comment:
It's fine removing the `carbon-dialog-full-screen__dialog--open` as I introduced it in the target branch to try to fix the scrolling issue